### PR TITLE
Cleanup NavView tests

### LINKED ISSUE #2468: Cleanup NavigationView tests
Currently there are a few tests which are not that helpful and should be cleaned up or removed:

- EnsureNoCrashesInHorizontalFlipMenuItemsSource <- Can this be removed or what is the description meaning
- EnsureNoCrashesInHorizontalFlipMenuItems <- Same as above
- MenuItemKeyboardInvokeTest <- Should probably be a test
- CanDoSelectionChangedOfItemTemplate <- `Should probably verify more

### PR DESCRIPTION
**Pull request by chingucoding**
## Description

Addresses the unhelpful tests addressed during #2396 and requested [here](https://github.com/microsoft/microsoft-ui-xaml/pull/2396#issuecomment-624954905) and [here](https://github.com/microsoft/microsoft-ui-xaml/pull/2396#issuecomment-630456714)
## Motivation and Context

Closes #2468
## How Has This Been Tested?

Ran the tests in question

### CODE REVIEW

**Comment by chingucoding** [comment]
Looks like the C++ compiler ran out of memory.

**Comment by ranjeshj** [comment]
> Looks like the C++ compiler ran out of memory.

@kmahone Is taking a look. [This PR](https://github.com/microsoft/microsoft-ui-xaml/pull/2787) will likely fix the issue.

File: dev/NavigationView/NavigationView_InteractionTests/FocusBehaviorTests.cs [hunk 1]
```diff
@@ -243,14 +243,24 @@ public void CanDoSelectionChangedOfItemTemplate()
         {
             using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", "NavigationView ItemTemplate Test" }))
             {
-                // Go to the navview items.
+                // Set focus inside page
+                var focusButton = FindElement.ByName("FocusAnchorButton");
+                focusButton.SetFocus();
```
Comment (StephenLPeters) [inline dev/NavigationView/NavigationView_InteractionTests/FocusBehaviorTests.cs:248]: Why is this now needed?

Comment (StephenLPeters) [reply dev/NavigationView/NavigationView_InteractionTests/FocusBehaviorTests.cs:248]: Ah i see now.

---
In reply to: [447890075](https://github.com/microsoft/microsoft-ui-xaml/pull/2771#discussion_r447890075) [](ancestors = 447890075)

**Pull request merged**